libs/blocks/youtube/youtube.js

```diff
@@ -19,6 +21,19 @@ class LiteYTEmbed extends HTMLElement {
     this.needsYTApiForAutoplay = navigator.vendor.includes('Apple') || this.isMobile;
   }
 
+  async fetchVideoTitle() {
+    if (this.playLabel !== LiteYTEmbed.defaultTitle) return null;
```

Comment:
I'm not sure I follow the logic between Vimeo and YouTube. On your [test page](https://adobe.sharepoint.com/:w:/r/sites/adobecom/_layouts/15/Doc.aspx?sourcedoc=%7B4F570997-5378-4427-83F5-3E5F88D554DE%7D&file=iframe-video-testing.docx&action=default&mobileredirect=true), Vimeo gets the fetched title, while YouTube doesn't. YouTube's remains "Youtube display text"; both seem to be authored in the same manner, a link with custom text. Am I missing something?